waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -945,20 +976,25 @@ proc subscribeToGroupEvents(ethClientUri: string, ethAccountAddress: Option[Addr
 
   proc startSubscription(web3: Web3) {.async, gcsafe.} =
     # subscribe to the MemberRegistered events
-    # TODO can do similarly for deletion events, though it is not yet supported
-    discard await subscribeToMemberRegistrations(web3, contractAddress, handler, blockNumber)
+    # TODO: can do similarly for deletion events, though it is not yet supported
+    # TODO: add block number for reconnection logic
+    discard await subscribeToMemberRegistrations(web3 = web3,
+                                                 contractAddress = contractAddress,
+                                                 handler = handler)
   
   await startSubscription(web3)
   web3.onDisconnect = proc() =
     debug "connection to ethereum node dropped", lastBlock = latestBlock
 
 
-  
-
-proc handleGroupUpdates*(rlnPeer: WakuRLNRelay, handler: RegistrationEventHandler) {.async, gcsafe.} =
+proc handleGroupUpdates*(rlnPeer: WakuRLNRelay) {.async, gcsafe.} =
   # mounts the supplied handler for the registration events emitting from the membership contract
```

Comment:
Do these comments still hold?

Reply:
Addressed in [582a6c3](https://github.com/status-im/nwaku/pull/1296/commits/582a6c322025d6d125f97ad434311170bb454c26)